packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -142,7 +142,11 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
 
```

Comment:
Can you please delete StorageTokenRefresh above then? Or do we want to keep it?

Reply:
Yeah.  If you don't see a use of that either, I will remove it.

Reply:
@christiango do you want this separate event to flag perf problems?  Or is it enough to get the information out of the perf event?

IMO, we should remove it.  I generally hate duplicate information.

Reply:
Let's remove it! I think the other event is sufficient, I am also a fan of less duplication

Reply:
done.